### DISCUSSION TITLE
macOS Support

### DISCUSSION THREAD

**Discussion by flowchartsman** [zealdocs/zeal#1576] [General]
Now that Dash has [changed its licensing model with version 7](https://blog.kapeli.com/dash-7), I expect a lot of people will be looking for a free alternative, so I'd like to open a new discussion here about macOS as a supported platform. There are at least a couple forks that look relatively close.

I don't condemn Kapeli for their choice and would like to avoid discussions on the merit of the project's license here. I am just on the fence about whether or not it offers enough over Zeal for my needs to justify a subscription, so I'd like to try evaluating Zeal for myself. I expect there are others in a similar boat, and it would be nice to have the opportunity to make that evaluation.

**Comment by Abdillah** [comment]
I'm going to use Zeal as the app, not its doc-set. I'm going to put our own docs to see. 
With that, Dash subscription is not a reasonable choice.

**Comment by trollixx** [comment]
Zeal can be built on macOS and should work. Might look very foreign and ugly though. Per our agreement with Dash, there will be no official builds of Zeal for macOS. Replacing all docsets is not something I have time and resources for. Paying Dash users benefit everyone else who gets docsets for free through Zeal.

I also don't have any Apple hardware, and barely find time to provide mediocre maintenance for Zeal on Linux and Windows. I am happy to accept reasonably looking pull requests that improve macOS support, but that's pretty much it.

**Reply by Abdillah** [reply]
True!
I've forked Zeal to add necessary bits for MacOS and confirmed that it builds.

github.com/domainesia/zeal

- Add pipeline for MacOS build
- Add CMake adjustment

Can I upstream them after I finished (the GH Action still failed)?

**Reply by trollixx** [reply]
I think currently Zeal builds on macOS correctly. If anything is missing PRs are welcome.
